src/machine/machine_stm32f407.go

```diff
@@ -7,44 +7,40 @@ package machine
 import (
 	"device/stm32"
 	"runtime/interrupt"
+	"unsafe"
 )
 
 func CPUFrequency() uint32 {
 	return 168000000
 }
 
+// Peripheral clock frequencies as set in runtime_stm32f407.go
+func apb1_Frequency() uint32 {
+	return CPUFrequency() / 4
+}
+
+func apb2_Frequency() uint32 {
+	return CPUFrequency() / 2
+}
+
```

Comment:
I've left these in for now as they'll come into play with the UART and SPI refactor for calculating baud rates.

Reply:
They should probably be named `apb1Frequency()` and `apb2Frequency()` to be more idiomatic function mames in Go.

But that is if they are left in at all. Seems like explicit inline  `(CPUFrequency() / 4)` is clearer than `apb1Frequency()` to me?

Reply:
I'm definitely ok with a rename, but hesitant to remove them, as they are referring to the peripheral bus speeds on the chip, as set in the `runtime` chip initialization code; ideally IMHO they'd go there. Otherwise for every peripheral the coder will need to refer back to the runtime chip init to determine what the current bus speeds are set to.
I agree that the explicit code makes clear what frequency is being used, but I feel it obscures why.

Reply:
Is this meant to be used in a future change? Because I can't find these functions being used anywhere in this PR.

Also, I agree with @deadprogram that these should be renamed to be more in Go style.

Reply:
yes, they show up in the UART and SPI refactors for calculating the baud rate those peripherals will end up with, and will probably appear in PWM when I get around to that.

Reply:
I think given the confusion these are causing in this PR, I'll take them out for now and reintroduce them with the UART refactor which I'll submit once this one is in.